### PR TITLE
Update meta-neuro version 1.0.0

### PR DESCRIPTION
**Pull request by bagari**
Hello,
My package meta merged at PR https://github.com/bioconda/bioconda-recipes/pull/51349, and it's listed at https://bioconda.github.io/recipes/meta/README.html#package-meta. However, I can't see it on the bioconda channel. Could you please help me with this?

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/meta/meta.yaml [hunk 1]
```diff
@@ -24,35 +23,28 @@ requirements:
...
-    - boost =1.82
-    - itk =5.3.0
-    - libitk =5.3.0
-    - libitk-devel =5.3.0
+    - setuptools
+    - libboost-python-devel
     - hdf5 >=1.14.2,<1.14.3.0a0
+    - itk ==5.3.0
+    - libitk ==5.3.0
+    - libitk-devel ==5.3.0
```
[the remark this answers is not in the document]
Comment (bagari) [reply recipes/meta/meta.yaml]: Incorporated these suggestions.

**Comment by bagari** [comment]
@BiocondaBot please add label

**Comment by bagari** [comment]
Hello,
I wonder if I need to change the build number from 0 to 1. But meta 1.0.0 is not available at the bioconda repo

**Comment by bagari** [comment]
I got an error when I tried to add support to py38-py311 and exclude py3.12. "failed with error: [Errno 28] No space left on device"

**Comment by bagari** [comment]
I have attached two screenshots of the storage before and after the test(before the test before_test 
and after the test:
after_test)

**Comment by bagari** [comment]
@BiocondaBot please update

**Comment by bagari** [comment]
@BiocondaBot rebuild

**Comment by bagari** [comment]
@BiocondaBot rebuild

File: recipes/meta/meta.yaml [hunk 2]
```diff
@@ -13,7 +12,8 @@
 
 build:
   number: 0
-  skip: true # [not py39]
+  # skip: true  # [py<39 or py>311]
+  skip: true  # [not py310]
```
[the remark this answers is not in the document]
Comment (bagari) [reply recipes/meta/meta.yaml:16]: I have it before, but I got "No space left on device" error

File: recipes/meta/meta.yaml [hunk 3]
```diff
@@ -13,7 +12,8 @@
 
 build:
   number: 0
```
[the remark this answers is not in the document]
Comment (bagari) [reply recipes/meta/meta.yaml]: updated build number to 1

**Comment by bagari** [comment]
@BiocondaBot please fetch artifacts

**Comment by bagari** [comment]
@BiocondaBot please fetch artifacts

**Comment by bagari** [comment]
I built meta package version 1.0.0 and supported OSX-64, OSX-ARM, and Linux-64. However, I cannot make all Python variants as I got an error saying, "No space left on device." Could you please help me to resolve this issue? I really appreciate any help you can provide.
@BiocondaBot please add label

**Comment by bagari** [comment]
@BiocondaBot please fetch artifacts

**Comment by bagari** [comment]
Hello, Could you please review and merge my "meta" version 1.0.0 package? I had an issue when it merged at PR https://github.com/bioconda/bioconda-recipes/pull/51349, but it was not available on the bioconda channel.
@BiocondaBot, please add label

**Comment by bgruening** [comment]
@bagari is there any chance you can change the name of the Bioconda package? The name `meta` is very generic, maybe something like `medial_tractography_analysis`?

**Comment by bagari** [comment]
[bgruening](https://github.com/bgruening) It's already available at https://bioconda.github.io/recipes/meta/README.html#package-meta. I could change it to `meta-neuro`. Will you be able to update the link?

**Comment by bgruening** [comment]
The link will be updated! Thanks for your understanding!

**Review by bgruening — APPROVED** [review]
Thanks a lot!

**Comment by bagari** [comment]
Thank you so much! [bgruening](https://github.com/bgruening)

**Comment by bgruening** [comment]
Thanks to you. I need to admin that I tried to read about this package and I only understood half of the words - crazy stuff!

**Comment by bagari** [comment]
@BiocondaBot please update

**Comment by bagari** [comment]
@BiocondaBot please fetch artifacts

**Pull request merged**